Example/Source/Component/SelectionCell/SelectionCell.swift

```diff
@@ -30,7 +30,7 @@ enum ExampleType {
     static let allValues: [ExampleType] = [.plainTableView, .playground, .stackView, .progressView]
 }
 
-final class SelectionCell: ControlBase<ExampleType, ExampleType> {
+final class SelectionCell: ButtonBase<ExampleType, ExampleType> {
```

Comment:
Why? Why is Reactant downgraded back to 1.1.0?

Reply:
Was the `ControlBase` released though?

Reply:
I have no memory of changing this.